client/components/payment-methods-list/payment-method.tsx

```diff
@@ -350,7 +350,7 @@ const PaymentMethod = ( {
 						) }
 						<a
 							// eslint-disable-next-line max-len
-							href="https://woo.com/document/woopayments/payment-methods/additional-payment-methods/#sofort-deprecation"
+							href="https://woo.com/document/woopayments/payment-methods/additional-payment-methods/#sofort-migration"
```

Comment:
What's the rationale for this change? Seems unrelated. 

> Fix sofort migration links - replaced the docs link from #sofort-deprecation to #sofort-migration.

I saw  ⏫  in PR description but didn't see a linked issue or explanation why – i.e. are there external links that use `-migration` that were broken? Or are we changing the wording to soften (`deprecation` is hard)?

Reply:
This issue is indeed unrelated. The old link https://woo.com/document/woopayments/payment-methods/additional-payment-methods/#sofort-deprecation doesn't work anymore. I think someone from the docs team renamed the Sofort section (and hence the internal link) recently, causing it to break here.

I noticed it by chance while testing this PR. I included the fix here as the code change is loosely related to what I intended to originally do - fix text on the Settings page.

Reply:
Updated the PR description with the reasoning for this change. Marking this thread as resolved. Thanks for the tip!